src/test/javascript/spec/component/learner-profile/component/course-learner-profile.component.spec.ts

```
@@ -0,0 +1,197 @@
+        expect(component.courseLearnerProfiles).toBe(profiles);
+    });
+
+    it('should select active course', () => {
+        let course = 1;
+        selectCourse(course);
+        let changeEvent = new Event('change', { bubbles: true, cancelable: false });
+        selector.dispatchEvent(changeEvent);
+        expect(component.activeCourse).toBe(course);
+    });
+
+    function setupUpdateTest(course: number, updateFn: Function, attribute: keyof CourseLearnerProfileDTO): CourseLearnerProfileDTO {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)